app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -59,6 +60,52 @@ def in_person_enrollment?(user)
         # WILLFIX: After LG-6872 and we have enrollment saved, reference enrollment instead.
         ProofingComponent.find_by(user: user)&.document_check == Idp::Constants::Vendors::USPS
       end
+
+      def usps_proofer
+        if IdentityConfig.store.usps_mock_fallback
+          UspsInPersonProofing::Mock::Proofer.new
+        else
+          UspsInPersonProofing::Proofer.new
+        end
+      end
+
+      def create_usps_enrollment(unique_id)
+        pii = user_session[:idv][:pii]
+        applicant = UspsInPersonProofing::Applicant.new(
+          {
+            unique_id: unique_id,
+            first_name: pii.first_name,
+            last_name: pii.last_name,
+            address: pii.address1,
+            # do we need address2?
+            city: pii.city,
+            state: pii.state,
+            zip_code: pii.zipcode,
+            email: 'no-reply@login.gov',
+          },
+        )
+        proofer = usps_proofer
+
+        response = proofer.request_enroll(applicant)
+        response['enrollmentCode']
```

Comment:
I'm planning on adding error handling in a follow-up PR

Reply:
WDYT of creating a Struct-like object to repesent the response here so we could do a direct property access `response.enrollment_code` rather than digging into a hash? Makes mocking responses easier as well